app/src/main/java/org/jellyfin/androidtv/util/sdk/BaseItemExtensions.kt

```diff
@@ -62,6 +62,20 @@ fun BaseItemDto.getProgramSubText(context: Context) = buildString {
 	// Add the channel name if set
 	channelName?.let { append(channelName, " - ") }
 
+	// Add season and episode info if available
+	if (parentIndexNumber != null && parentIndexNumber != 0) {
+		append(context.getString(R.string.lbl_season_number, parentIndexNumber))
+		append(" ")
```

Comment:
The other place that adds a space does it like this

```suggestion
		append(context.getString(R.string.lbl_season_number, parentIndexNumber), " ")
```